Main README does not mention the sphinx-tab requirement (#363)
ros2/ros2_documentation: README.md

### Prerequisites

> You must have sphinx and a couple of required sphinx extensions installed, preferrably using pip:

Reviewer: ```suggestion
You must have sphinx and the required sphinx extensions installed, preferably using pip:
```

Author: Agreed. Changed and updated branch with your suggested change. 

Anything else?
